test/helpers.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) F5, Inc.
```

Comment:
We are using `require.NoError`, `t.Fail` and `assert.Fail` in this file when checking errors. We should pick one way and be consistent.

Reply:
that was updated based on previous comments